Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace]
 resolver = "2"
 
-package.version = "0.1.0"
+package.version = "0.1.3"
```

Comment:
Bumped version from 0.1.0 to 0.1.3, to keep the same version number as the corresponding ndc-spec.

Is this a good idea, should we do something else instead? The thinking is having a single number wil be less confusing.